calyx-py/test/correctness/pifo_tree.futil

```diff
@@ -0,0 +1,1068 @@
+import "primitives/core.futil";
```

Comment:
Do we want this file committed? It's rather long so if we're testing code generation, maybe we can generate a smaller file? If this is the smallest we can get then no worries.

Reply:
Whoops, thanks. Nope, no need to commit it. I just generated for fun, and to show Tobias :)